ansys/dpf/core/dpf_operator.py

```diff
@@ -144,14 +163,16 @@ def connect(self, pin, inpt, pin_out=0):
         request = operator_pb2.UpdateRequest()
         request.op.CopyFrom(self._message)
         request.pin = pin
-        _fillConnectionRequestMessage(request, inpt, pin_out)
+        tmp = _fillConnectionRequestMessage(request, inpt, self._server, pin_out)
```

Comment:
Not sure if `tmp` is necessary here.

Reply:
it's necessary (but very ugly). I did that to make sure that tmp is not collected by the garbage collector